lab-steven/app/services/player-service.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+require('angular')
+.module('ngAdventure')
+.factory('playerService', ['$q', '$log', 'mapService', function($q, $log, mapService){
+  $log.debug('playerService');
+
+  let service = {};
+  let turn = 0;
+  let player = service.player = {
+    name: 'player 1',
+    location: 'town',
+    hp: 8,
```

Comment:
what happens when your hp falls below 8? when i played your game, there was no way to "lose". hp just went negative. it does function properly, though!